internal/regex/regex_oniguruma.go

```diff
@@ -1,3 +1,5 @@
+// +build mysql_go_onigurama
```

Comment:
enry has just `oniguruma` afaik

Reply:
It was for symmetry with the previous tag, but will change it to onigurama.

Reply:
PS: done.